docs/docs/dependency-specification.md

```diff
@@ -98,6 +98,12 @@ flask = { git = "https://github.com/pallets/flask.git", rev = "38eb5d3b" }
 numpy = { git = "https://github.com/numpy/numpy.git", tag = "v0.13.2" }
 ```
 
+To use ssh connection (for example for private reopsitories) simply follow the standard git syntax:
```

Comment:
```suggestion
To use an SSH connection, for e.g. in the case of private repositories, simply follow the standard git syntax:
```

Reply:
thank you. good improvement, only will slightly change `for e.g.`, since `e.g.` translates to `for example`.

```suggestion
To use an SSH connection,  for example in the case of  private repositories, simply follow the standard git syntax:
```